core/modules/replica/ConfigApp.cc

```diff
@@ -0,0 +1,911 @@
+        _dumpDbShowPassword
+    );
+
+    parser().command("CONFIG_INIT_FILE").required(
+        "format",
+        "The format of the initialization file to be produced with this option."
+        " Allowed values: MYSQL, INI",
+        _format,
+        vector<string>({"MYSQL", "INI"})
+    );
+
+    parser().command("UPDATE_WORKER").required(
+        "worker",
+        "The name of a worker to be updated",
+        _workerInfo.name
+    ).option(
+        "worker-service-host",
+        "The new DNS name or an IP address where the worker runs",
+        _workerInfo.svcHost
+    ).option(
+        "worker-service-port",
+        "The port number of the worker service",
+        _workerInfo.svcPort
+    ).option(
+        "worker-fs-host",
+        "The new DNS name or an IP address where the worker's File Server runs",
+        _workerInfo.fsHost
+    ).option(
+        "worker-fs-port",
+        "The port number of the worker's File Server",
+        _workerInfo.fsPort
+    ).option(
+        "worker-data-dir",
+        "The data directory of the worker",
+        _workerInfo.dataDir
+    ).flag(
+        "worker-enable",
+        "Enable the worker. ATTENTION: this flag can't be used together with flag --worker-disable",
+        _workerEnable
+    ).flag(
+        "worker-disable",
+        "Disable the worker. ATTENTION: this flag can't be used together with flag --worker-enable",
+        _workerDisable
+    ).flag(
+        "worker-read-only",
+        "Turn the worker into the read-only mode. ATTENTION: this flag can't be"
+        " used together with flag --worker-read-write",
+        _workerReadOnly
+    ).flag(
+        "worker-read-write",
+        "Turn the worker into the read-write mode. ATTENTION: this flag can't be"
+        " used together with flag --worker-read-only",
+        _workerReadWrite
+    );
+
+    parser().command("ADD_WORKER").required(
+        "worker",
+        "The name of a worker to be added",
+        _workerInfo.name
+    ).required(
+        "service-host",
+        "The DNS name or an IP address where the worker runs",
+        _workerInfo.svcHost
+    ).required(
+        "service-port",
+        "The port number of the worker service",
+        _workerInfo.svcPort
+    ).required(
+        "fs-host",
+        "The DNS name or an IP address where the worker's File Server runs",
+        _workerInfo.fsHost
+    ).required(
+        "fs-port",
+        "The port number of the worker's File Server",
+        _workerInfo.fsPort
+    ).required(
+        "data-dir",
+        "The data directory of the worker",
+        _workerInfo.dataDir
+    ).required(
+        "enabled",
+        "Set to '0' if the worker is turned into disabled mode upon creation",
+        _workerInfo.isEnabled
+    ).required(
+        "read-only",
+        "Set to '0' if the worker is NOT turned into the read-only mode upon creation",
+        _workerInfo.isReadOnly
+    );
```

Comment:
Not quite. The command line syntax of these two command is fundamentally different. IN case of the first one all parameters are mandatory and positional:
```
ADD_WORKER <worker> <service-host> <service-port> <fs-host> <fs-port> <data-dir> <enabled> <read-only>
```
Meanwhile, for the second one, these are options with values or flags. The only exception is the name of a worker:
```
UPDATE_WORKER UPDATE_WORKER  <worker> 
  --worker-data-dir=[<value>]
  --worker-fs-host=[<value>]
  --worker-fs-port=[<value>]
  --worker-service-host=[<value>]
  --worker-service-port=[<value>]
  --worker-disable
  --worker-enable
  --worker-read-only
  --worker-read-write
```
The last command allows for selective modification of attributes of some known replication worker.